src/main/java/book/store/service/impl/ShoppingCartManagerServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package book.store.service.impl;
+
+import book.store.model.ShoppingCart;
+import book.store.model.User;
+import book.store.repository.shoppingcart.ShoppingCartRepository;
+import book.store.service.ShoppingCartManagerService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ShoppingCartManagerServiceImpl implements ShoppingCartManagerService {
+    private final ShoppingCartRepository cartRepository;
+
+    public ShoppingCart createShoppingCart(User user) {
```

Comment:
this method should be in ShoppingCartService
```suggestion
    public void createShoppingCart(User user) {
```